addons/jam/CfgWeapons.hpp

```diff
@@ -31,9 +31,20 @@ class CfgWeapons {
         };
     };
 
+    class arifle_Katiba_Base_F : Rifle_Base_F {
+        magazineWell[] = {"CBA_65x39_Katiba"};
+    };
+
+    class arifle_ARX_base_F : Rifle_Base_F {
+        magazineWell[] = {"CBA_65x39_Katiba"};
+    };
+
     class EBR_base_F: Rifle_Long_Base_F {
         magazineWell[] = {"CBA_762x51_M14"};
     };
+    class DMR_03_base_F: Rifle_Long_Base_F {
+        magazineWell[] = {"CBA_762x51_M14"};
```

Comment:
Why ? What weapon is it and does it use M14 mags ? Why not AR10 mags ? Or FAL ? Or G3 ?

Reply:
It's a SIG 556 ("Mk 1 EMR") chambered for 7.62x51. It supports the same magazines as vanilla's M14 and HK417.

To mimic Vanilla behaviour, I could add the same magazine wells that are used by other vanilla-DMRs.

Reply:
Again, an exercise of imagination, because SIG 556 doesn't exist in 7.62x51. Maybe a SIG 716 is a closer match ? In that case it should take AR10 mags, not M14. Yeah, vanilla has even the HK417 use that same mag, but let's keep JAM/magwell stuff realistic, not based off vanilla.

Reply:
I guess it's kind of a hybrid of both weapons.

Changing it to AR10 mags seems fair to me with the SIG 716 accepting them, but if so, vanilla mags fitting into it should also be categorized as AR10 mags.

I agree with you on keeping stuff realistic, but as long as BI doesn't introduce a lot of new mags, let's also make sure that vanilla weapons are compatible with modded magazines based on vanilla magazines that fit into it.

Reply:
Community mods add the AR10 mags and they will fit. Ignore the stray vanilla mod that fits all.
If you add the M14 magwell you will end up with a long array of mags that fit when they shouldn't.
If you put a better match magwell you only end up with the one weird vanilla mag case.
Just my 2c.